include/ivmgr.h

```diff
@@ -34,7 +34,7 @@
 #include <qstring.h>
 #include <qwidget.h>
 #include "material.h"
-
+#include "ivmgrHeadless.h"
 #include <Inventor/Qt/viewers/SoQtExaminerViewer.h>
```

Comment:
You are right, of course. It's not needed, this must have snug in when I was testing something. Thanks for pointing it out, well spotted!